cmd/soroban-cli/src/contract/invoke.rs

```diff
@@ -41,9 +41,6 @@ pub struct Cmd {
     /// Argument to pass to the function
     #[clap(long = "arg", value_name = "arg", multiple = true)]
     args: Vec<String>,
-    /// Argument to pass to the function (base64-encoded xdr)
-    #[clap(long = "arg-xdr", value_name = "arg-xdr", multiple = true)]
-    args_xdr: Vec<String>,
```

Comment:
I worry about this, as there are some args (for example `Identifier`s), which are super difficult to pass via the `--arg` flag.

Reply:
^ is this true @tyvdh ?

Reply:
I'll defer to @willemneal but if his PR around arg improvements goes live we may not need it. I definitely want to ensure all possible args are able to be supported but I'm under the impression his PR will do that

https://github.com/stellar/soroban-tools/pull/307
https://github.com/stellar/soroban-tools/pull/284

Reply:
Ok then. #307 looks good as well, so let's do it.

Reply:
Pro tip, @tyvdh: list PRs in bulleted lists and GitHub expands them to show status and title. You could edit your comment to just add `- ` at the beginning of each of your PR lines.

- #307
- #284